src/Client.php

```diff
@@ -42,7 +43,11 @@ public function getIndexer(): Indexer
         return $this->indexer;
     }
 
-    public function getIndex($name): Index
+    /**
+     * @param string $name
+     * @return ElasticaIndex|Index
```

Comment:
This phpDoc is wrong, the code is very explicit here, we return an Elastically Index object, 
could you tell us more about this change?

Reply:
Could we add PHP Type hint everywhere instead?